### ISSUE TITLE
After calling the document conversion interface to complete the document conversion, error 403 is reported after the URL request is returned

### ISSUE THREAD

**Issue by cv-msan** [ONLYOFFICE/DocumentServer#2506]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Windows Server

### Version information

7.5

### Expected Behavior

After calling the document conversion interface to complete the document conversion, requesting the returned URL and then accessing the browser to request the download of the document, but returning 403
![image](https://github.com/ONLYOFFICE/DocumentServer/assets/76542419/32ade1c7-af02-4d86-933b-cc072ce679af)
![image](https://github.com/ONLYOFFICE/DocumentServer/assets/76542419/fd6bcea5-ac3c-4c19-9ba6-e387a4efea14)

### Actual Behavior

After calling the document conversion interface to complete the document conversion, request the returned URL and then visit the browser to request the download of the document. The download should be successful

**Comment by l8556** [comment]
Hi @cv-msan !
You need to remove `amp;` from the resulting URL.
Since you are using postman, go to the `Preview` tab and copy the URL from there.

**Comment by cv-msan** [comment]
@l8556 Yes, I discovered this issue around 6 o'clock last night and indeed just remove the amp. Sorry, I forgot to close the issue
